jobs/build-arch.Jenkinsfile

```diff
@@ -187,6 +193,17 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
                     --url s3://${s3_stream_dir}/builds \
                     --aws-config-file \${AWS_BUILD_UPLOAD_CONFIG}
                 """)
+                // Also fetch the x86_64 lockfile for the build we're
+                // complementing, if any. Usually, this is the same build we
+                // just fetched above, but not necessarily.
+                pipeutils.shwrapWithAWSBuildUploadCredentials("""
+                cosa buildfetch --arch=x86_64 \
+                    --build ${newBuildID} \
+                    --file manifest-lock.generated.x86_64.json \
+                    --url s3://${s3_stream_dir}/builds \
+                    --aws-config-file \${AWS_BUILD_UPLOAD_CONFIG}
+                cosa shell -- ls -lR builds/
```

Comment:
Do we intend to remove the `cosa shell -- ls -lR builds/` before merging this?

Reply:
Yeah, leftover from debugging. :)